### DISCUSSION TITLE
i3's config: Open Terminal & cat file?

### DISCUSSION THREAD

**Discussion by och4** [i3/i3#5626] [Q&A]
How do I get i3's config to open a new terminal and cat a file within that terminal? The following isn't working for me. My googl-fu  is weak and I can't resolve this on my own.

```
# This line is broken. catting the file externally works, but the whole line doesn't.
exec --no-startup-id sh -c 'sleep 2; i3-msg "workspace $ws11"; exec gnome-terminal -e "cat /tmp/stdout"'

# This line works as intended. 
exec --no-startup-id sh -c 'sleep 3; i3-msg "workspace $ws12"; exec gnome-terminal -e "journalctl -f"'
```

**Comment by orestisfl** [comment]
From `man gnome-terminal` the syntax you are looking for is
```shell
gnome-terminal -- cat /tmp/stdout
```

See https://askubuntu.com/a/630779 for holding the terminal open or just try

```shell
gnome-terminal -- sh -c 'cat /tmp/stdout; sleep 100d'
```

**Reply by och4** [reply]
I've tried testing this in a plain terminal window, but neither of these worked. I also tried throwing them into my i3 config anyway, but of course, it didn't work either. When I tried entering these lines into my plain terminal window, something flashes on my screen like it's trying to open a new terminal, but it never appears. I went digging through the man pages, but that didn't help. Any ideas?
